ex5.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def add_vector(matrix: np.ndarray, vector: np.ndarray) -> np.ndarray:
+    """
+       The function add the vector for each row in the given matrix.
+       @param matrix: The given matrix  
+       @param vector: the given vector 
+       @return: tuple contains the row and col of the matrix.
+       """
+    result = np.empty_like(matrix)
+    for index in range(0, matrix.shape[0]):
+        result[index, :] = matrix[index, :] + vector
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html